.github/workflows/docker.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2024 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+# GitHub Actions workflow to publish Docker images.
+name: "Docker"
+on:
+  push:
```

Comment:
should we do this on every push? I would like to.  push a tag up per-commit (for the push)